docs/iris/src/whatsnew/2.0a0.rst

```diff
@@ -15,31 +15,33 @@ Iris 2.0a0 Features
 
 .. admonition:: Showcase Feature - Dask Integration
 
-  The use of `biggus`_ to provide support for *virtual array* and
+  The use of `biggus`_ to provide support for *virtual arrays* and
   *lazy evaluation* within iris has been replaced with `dask`_.
 
-  This is a major feature enhancement, which extends the concept of *lazy data*
-  available in a :class:`~iris.cube.Cube` with a `dask Array`_ to an
-  :class:`~iris.coords.AuxCoord`, :class:`~iris.coords.CellMeasure` and
-  :class:`~iris.aux_factory.AuxCoordFactory`. Building upon `dask`_ allows
-  iris to easily leverage its rich functionality and capabilities. Most
-  notably, iris can employ either of its *threaded*, *multi-processor* or
-  *distributed* `schedulers`_ in order to fully exploit the available compute
-  resource. Also, you can use `dask Bags`_ or `dask Delayed`_ to easily
-  parallelise your algorithms.
+  In addition the concept of *lazy data*, already used for the
+  :class:`~iris.cube.Cube` data component, has now been extended to the data arrays in
+  :class:`~iris.coords.Coord` s and :class:`~iris.aux_factory.AuxCoordFactory` s.
+
+  This is a major feature enhancement, because basing Iris data in `dask`_
+  allows iris to easily leverage its rich functionality and capabilities.
+  In particular, dask's *threaded*, *multi-processor* or *distributed*
+  `schedulers`_ can be used in order to fully exploit the available compute
+  resource.
+
+  For fuller details, see :doc:`"Real and Lazy Data" </whitepapers/real_and_lazy_data>`.
 
 * Changes to the :class:`iris.cube.Cube`:
 
   * A :class:`~iris.cube.Cube` now accepts the *new* kwargs :data:`dtype` and
     :data:`fill_value`.
 
-  * The :meth:`~iris.cube.Cube.copy` method now accepts the *new* kwargs
+  * The :attr:`~iris.cube.Cube.copy` method now accepts the *new* kwargs
```

Comment:
@pp-mo This is a method not an attribute, so I think `:meth:` still stands here ...